consensus/e2e_tests/utils_test.go

```diff
@@ -414,6 +423,7 @@ func baseP2PMock(t *testing.T, eventsChannel modules.EventsChannel) *mockModules
 			eventsChannel <- e
 		}).
 		AnyTimes()
+	// CONSIDER: Adding a check to not to send message to itself
```

Comment:
Look in our `Makefile` to see what our kewyrods are

```suggestion
	// CONSIDERATION: Adding a check to not to send message to itself
```